example/lib/async/main.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:example/async/ui/app.dart';
```

Comment:
I appreciate the example app from a more "real world usage" point of view, but it defeats the purpose of the example of allowing people to understand what a package API looks like when used.